src/api/Feed.js

```diff
@@ -138,9 +158,9 @@ class Feed extends Base {
 
         this.id = id;
         this.hasError = false;
-        const versionsPromise = this.fetchVersions();
-        const commentsPromise = this.fetchComments(permissions);
-        const tasksPromise = this.fetchTasks();
+        const versionsPromise = this.fetchVersions(onError);
```

Comment:
Should `onError` instead be a `.catch` clause of `Promise.all` below? That way we can avoid passing the callback around and just throw errors from the child methods.

Reply:
The problem with that is that the calls below are not actually rejecting when they fail, just resolving undefined due to displaying of partial data if possible.

If I were to throw and added a `.catch`, the `.then` would never be executed